guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: guide
+title: Payments with Payjoin
+description: <>
+nav_order: 5
+parent: Case studies
+permalink: /guide/case-studies/payjoin-case-study/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-preview.jpg
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: <>
+
+-->
+
+<!-----
+Conversion notes:
+
+* Docs to Markdown version 1.0β34
+* Sun Mar 05 2023 13:50:07 GMT-0800 (PST)
+* Source doc: [restructure] Payments with Payjoin: A Case Study
+* Tables are currently converted to HTML tables.
+* This document has images: check for >>>>>  gd2md-html alert:  inline image link in generated source and store images to your server. NOTE: Images in exported zip file from Google Docs may not appear in  the same order as they do in your doc. Please check the images!
+
+----->
+
+# Payments with Payjoin: A Case Study
+{:.no_toc}
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+## Introduction
+
+Payjoin transactions can be a powerful tool providing privacy, fee-savings and UTXO consolidation benefits to users. They also present challenges and trade-offs due to their synchronous, multi-party nature.
```

Comment:
```suggestion
Payjoin transactions can be a powerful tool providing privacy, fee-savings and UTXO consolidation benefits to users. They also present challenges and trade-offs due to their interactive nature.
```

I think all but self-spend transactions  are multi-party. They key with payjoin is that  it is *interactive*. Synchronicity may be considered a detail.